packages/checkout/widgets-lib/src/views/error/serviceTypes.ts

```diff
@@ -1,3 +1,4 @@
 export enum ServiceType {
   SWAP = 'swap',
+  SANCTION = 'sanction',
```

Comment:
Not particularly happy with naming here. Want to re-use ServiceUnavailable view though